### ISSUE TITLE
Parent definition "...\UniqidNamer" does not exist

### ISSUE THREAD

**Issue by AxelBriche** [dustin10/VichUploaderBundle#848]
```
    mappings:
        car_images:
            uri_prefix: '%cars_images_path%'
            upload_destination: '%kernel.project_dir%/public/%cars_images_path%'
            namer: Vich\UploaderBundle\Naming\UniqidNamer

```
**The error:**
Service "Vich\UploaderBundle\Naming\UniqidNamer.car_images": Parent definition "Vich\UploaderBundle\Naming\UniqidNamer" does not exist.

**Comment by AxelBriche** [comment]
Fixed, work with service name directly:
namer:  vich_uploader.namer_uniqid

**Comment by garak** [comment]
That's not the intended way, service name are internally deprecated and will be removed in the next major version

**Comment by garak** [comment]
It should be fixed in the latest release, `1.8.2`
